infrastructure/httpapi/handlers/validators.go

```diff
@@ -285,14 +303,56 @@ func (handler *Validators) getAverageBlockTime() (*big.Float, error) {
 		}
 	}
 
-	totalBlockTimeMilliSecond := new(big.Float).Quo(
-		new(big.Float).SetInt(totalBlockTime),
-		new(big.Float).SetInt64(int64(1000000)),
-	)
-	averageBlockTimeMilliSecond := new(big.Float).Quo(
-		totalBlockTimeMilliSecond,
-		new(big.Float).SetInt(totalBlockCount),
-	)
+	nRecentBlocks := big.NewInt(nRecentBlocksInInt)
+
+	isTotalBlockCountExcess := (totalBlockCount.Cmp(nRecentBlocks) == 1)
```

Comment:
`isTotalBlockCountExcess` is probably not self-explanatory enough, what does it mean by excess? I will probably name it `hasNBlocksSinceGenesis`

Reply:
Agree! Will update the name.